ringpop.go

```diff
@@ -96,6 +96,25 @@ func mergeDefault(opts *Options) *Options {
 	return opts
 }
 
+type RingpopInterface interface {
```

Comment:
Best case would be to call it `Ringpop` and hide the struct behind a lower cased name so to not export the actual struct.